var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -171,20 +158,34 @@ def configure(self):
                 fh.write('\n'.join(makefile_inc))
 
     def install(self, spec, prefix):
-        self.configure()
-
         targets = ['scotch']
         if '+mpi' in self.spec:
             targets.append('ptscotch')
 
-        if '+esmumps' in self.spec:
-            targets.append('esmumps')
-            if '+mpi' in self.spec:
-                targets.append('ptesmumps')
+        if self.spec.version >= Version('6.0.0'):
```

Comment:
This change (in combination with the removal of the `patch:@6.0.0` function) effectively removes support for the `+esmumps` variant for all `@5` versions of `scotch`.  Why did you decide to remove this support?  Were you having problems compiling or linking against the `esmumps` and `ptesmumps` libraries output by these versions?